CHANGES.rst

```diff
@@ -90,6 +92,8 @@ Internal Changes
 - Add additional signals related to videoconferences and their event links (:pr:`6475`)
 - Videoconference plugins now need to implement a ``delete_room`` method (:pr:`6475`)
 - Support translator comments when extracting translatable strings (:pr:`6620`)
+- ``renderAsFieldset`` option in the registration field registry can now be a function that
+  returns a boolean (thanks :user:`foxbunny`)
```

Comment:
Not sure if this really needs a changelog entry, but I guess someone might be using it in a plugin so no harm in having it ;)

```suggestion
- ``renderAsFieldset`` option in the registration field registry can now be a function that
  returns a boolean (:pr:`6621`, thanks :user:`foxbunny`)
```